libs/platform/ebpf_extension.c

```diff
@@ -132,10 +134,15 @@ _ebpf_extension_client_attach_provider(
 
     local_client_binding_context->provider_is_attached = NT_SUCCESS(status);
 
-    if (NT_SUCCESS(status))
-        _ebpf_extension_client_notify_change(local_client_context, local_client_binding_context);
-    else
+    if (NT_SUCCESS(status)) {
+        ebpf_result_t result = _ebpf_extension_client_notify_change(local_client_context, local_client_binding_context);
+        if (result != EBPF_SUCCESS) {
+            status = STATUS_UNSUCCESSFUL;
```

Comment:
Curious why we're not logging this failure, when we're logging some other failure just a couple of lines below.